MapboxNavigation/RouteVoiceController.swift

```diff
@@ -369,7 +369,7 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         }
         
         // Only localized languages will have a proper fallback voice
-        utterance.voice = AVSpeechSynthesisVoice(language: Bundle.main.preferredLocalizations.first)
+        utterance.voice = AVSpeechSynthesisVoice(language: Locale.preferredLanguages.first!)
```

Comment:
@1ec5 how can we get even more specific localization than just `en`. If I set my region to `en-us`, `Bundle.main.preferredLocalizations.first` will only return `en`.